packages/client/src/index.ts

```diff
@@ -89,17 +83,14 @@ export class LogtoClient {
       prompt: 'consent',
       code_challenge: codeChallenge,
       code_challenge_method: 'S256',
+      redirect_uri: redirectUri,
     });
     return [url, codeVerifier];
   }
 
-  public async handleLoginCallback(codeVerifier: string, code: string) {
+  public async handleLoginCallback(redirectUri: string, codeVerifier: string, code: string) {
     const client = this.getClient();
-    const tokenset = await client.callback(
-      this.redirectUris[0],
-      { code },
-      { code_verifier: codeVerifier }
-    );
+    const tokenset = await client.callback(redirectUri, { code }, { code_verifier: codeVerifier });
```

Comment:
should be `tokenSet` per [doc](https://github.com/panva/node-openid-client/blob/main/docs/README.md#clientcallbackredirecturi-parameters-checks-extras):

> Returns: `Promise<TokenSet>` Parsed token endpoint response as a TokenSet.